docs/podman-images.1.md

```diff
@@ -11,6 +11,10 @@ Displays locally stored images, their names, and their IDs.
 
 ## OPTIONS
 
+**--all, -a**
+
+Show all images (by default filter out the intermediate image layers). The default is false.
```

Comment:
Should say this is a no-op here

Reply:
Well I just witnessed a lot of progress @umohnani8 is making to make --cacheing work in buildah.